ci/github/integration-testing/director-v2.bash

```diff
@@ -23,8 +23,7 @@ test() {
   echo "testing in services/director-v2/tests/integration/$1"
   pytest --cov=simcore_service_director_v2 --durations=10 --cov-append \
     --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
-    -vvv \
-    -v -m "not travis" "services/director-v2/tests/integration/$1"
+    -v -m "not travis" "services/director-v2/tests/integration/$1" --log-level=DEBUG
```

Comment:
you said  in one of your comments that you removed it. I still see it? forgotten?

Reply:
It looks similar to other tests now. Not sure if something else needs to be removed.